sktwocnt/lib/twocnt.f90

```diff
@@ -217,53 +213,58 @@ subroutine get_twocenter_integrals(inp, imap, skham, skover)
     !! number of radial and angular integration abscissas
     integer :: nRad, nAng
 
-    if (inp%iXC == 5) then
+    if (inp%iXC == 1) then
```

Comment:
If you changed this block any way, could you turn it please into a type based enumeration?
```
type :: TXcFunctionalsEnum
  integer :: LDA_PW91 = 1
  ...
end type TXcFunctionalsEnum

type(TXcFunctionalsEnum), parameter :: xcFunctionals = TXcFunctionalsEnum()

if (inp%iXC == xcFunctionals%LDA_PW91) then
```